API Project/src/main/java/com/bounswe/services/CulturalHeritageService.java

```diff
@@ -35,5 +35,10 @@ public void save(CulturalHeritage culturalHeritage) {
     while(it.hasNext()) culturalHeritages.add(it.next());
     return culturalHeritages;
   }
+
+  public void delete(CulturalHeritage culturalHeritage){
+	this.culturalHeritageRepository.delete(culturalHeritage);
```

Comment:
This won't work since delete method wants an Long ID. Please run `gradle bootRun` on your console and test the webserver yourself, and ensure you are able to delete a cultural heritage item.